### PR TITLE
V0.4.0rc3

### PR DESCRIPTION
**Pull request by mattwthompson**
Compare against commit

f596f6ce3c5067c96f2254bfd55839835247f353

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mattwthompson** [comment]
```
$ curl -sL https://github.com/openforcefield/openff-evaluator/archive/v0.4.0rc3.tar.gz | openssl sha256 | tail -n 1 | cut -d" " -f2
543201c5e814967a86b50213136110a081912c78a872ced18eb2d03b6564f460

**Comment by mattwthompson** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -35,7 +33,7 @@ outputs:
         - mdtraj >=1.9.3
         - dask >=2.7.0
         - distributed >=2.7.0
-        - dask-jobqueue >=0.7.0
+        - dask-jobqueue >=0.8.0
```
Comment (mattwthompson) [inline recipe/meta.yaml:36]: This is an actual change between versions https://github.com/openforcefield/openff-evaluator/pull/467 / https://github.com/openforcefield/openff-evaluator/pull/467/commits/b4a73e6d574cac28d9cf7b0062e4c8218b51d4b0

**Review by j-wags — APPROVED** [review]
LGTM!

**Comment by mattwthompson** [comment]
Thanks1

**Pull request merged**